files/startup-scripts/custom-config-updates.cli

```diff
@@ -18,4 +18,4 @@ batch
 
 run-batch
 
-stop-embedded-server
+stop-embedded-server	
```

Comment:
Nitpick: Extraneous whitespace (you should be able to set your editor to strip trailing whitespace on save)